src/web/releases.rs

```diff
@@ -413,39 +363,35 @@ pub fn releases_failures_by_stars_handler(req: &mut Request) -> IronResult<Respo
     releases_handler(req, ReleaseType::Failures)
 }
 
-pub fn author_handler(req: &mut Request) -> IronResult<Response> {
+pub fn owner_handler(req: &mut Request) -> IronResult<Response> {
     let router = extension!(req, Router);
     // page number of releases
     let page_number: i64 = router
         .find("page")
         .and_then(|page_num| page_num.parse().ok())
         .unwrap_or(1);
-    let author = router
-        .find("author")
-        // TODO: Accurate error here, the author wasn't provided
-        .ok_or(Nope::CrateNotFound)?;
+    let owner = router.find("owner").ok_or(Nope::OwnerNameNotProvided)?;
```

Comment:
This is how `owner_handler` is used in the router: https://github.com/rust-lang/docs.rs/blob/63647614a4b12d06b1b8ddf344e1cc439db05b8c/src/web/routes.rs#L44-L45
That should always have an `owner` field, or otherwise this function wouldn't be called. So I think this can just be an `unwrap()` and you can remove the `OwnerNameNotProvided` variant.

Reply:
Removed.